src/IconRenderer.vala

```diff
@@ -197,16 +197,21 @@ namespace Marlin {
             style_context.restore ();
 
             /* Do not show selection helpers or emblems for very small icons */
-            if ((selected || prelit) && file != drop_file) {
-                helper_size = Marlin.IconSize.LARGE_EMBLEM > (int.max (pixbuf.get_width (), pixbuf.get_height ()) / icon_scale) / 2 ?
-                              Marlin.IconSize.EMBLEM : Marlin.IconSize.LARGE_EMBLEM;
+            if ((focused || selected || prelit) && file != drop_file) {
+                if (focused &&!(selected || prelit)) {
+                    /* Always use small helper to show focus (use smaller?) to minimise obscuration of icon*/
```

Comment:
This kind of looks like a bug imo since the icon size changes as soon as you move the cursor or focus

Reply:
Yes, I was not sure about that. The reason I used a small helper to indicate focus was because the linked issue objected to the appearance of the icon changing when focused but not selected.  The small helper is less likely to partially obscure the icon.  However, it would be easy to have the helper stay the same size.  I would consider this a temporary fix pending a CSS fix (if possible).

I'll see if I can get a similar effect programmatically.